includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -258,7 +258,7 @@ private function getWikisSection() {
 						'type' => 'translatable-text',
 						'key' => 'global-navigation-brandlink-wikis-explore'
 					],
-					'href' => $this->getHref( 'explore-wikis' ),
+					'href' => $this->getHref( 'fan-communities' ),
```

Comment:
This is the link used for explore wikis. It makes sense for me, what do you think?